ffi/capi/src/units_converter.rs

```diff
@@ -69,13 +70,30 @@ pub mod ffi {
     pub struct ICU4XMeasureUnitParser<'a>(pub MeasureUnitParser<'a>);
 
     impl<'a> ICU4XMeasureUnitParser<'a> {
+        /// Parses the CLDR unit identifier (e.g. `meter-per-square-second`) and returns the corresponding [`ICU4XMeasureUnit`].
+        /// Returns an error if the unit identifier is not valid or the `unit_id` is not a valid UTF-8 string.
```

Comment:
This includes the other case. Both versions of the method should have the same docs, as some languages use one, and some use the other.
```suggestion
        /// Returns an error if the unit identifier is not valid.
```